scripts/android/build_decred.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+. ./config.sh
+CW_DECRED_DIR=${WORKDIR}/cake_wallet/cw_decred
+LIBWALLET_PATH="${WORKDIR}/decred/libwallet"
+LIBWALLET_URL="https://github.com/itswisdomagain/libwallet.git"
+
+mkdir -p $LIBWALLET_PATH
+git clone $LIBWALLET_URL $LIBWALLET_PATH --branch cgo
+cd $LIBWALLET_PATH
+
```

Comment:
I have to do this to build. Idk if it messes up other peoples builds but just thought I'd suggest. Taken from https://pub.dev/packages/dart_tinydtls

```suggestion

export CPATH="$(clang -v 2>&1 | grep "Selected GCC installation" | rev | cut -d' ' -f1 | rev)/include"

```

Reply:
What error do you get without it?

Reply:
I've seen it from the link you shared. I don't think this will cause any problems for anyone, will commit.